src/envoy/http/backend_auth/filter_config.h

```diff
@@ -27,9 +27,9 @@ namespace backend_auth {
  * All stats for the backend auth filter. @see stats_macros.h
  */
 #define ALL_BACKEND_AUTH_FILTER_STATS(COUNTER) \
-  COUNTER(denied_by_no_operation)              \
+  COUNTER(denied_by_no_route)                  \
   COUNTER(denied_by_no_token)                  \
-  COUNTER(allowed_by_no_configured_rules)      \
+  COUNTER(allowed_by_disable_auth)             \
```

Comment:
Not sure if renaming counters is OK for API Gateway, may break their monitoring. Let's discuss with them.

Either way, I don't know if `disable_auth` is a good name here, that's service config specific. How about `allowed_by_auth_not_required`

Reply:
Done